src/openloco/input.h

```diff
@@ -85,6 +85,9 @@ namespace openloco::input
     void cancel_tool();
     void cancel_tool(ui::WindowType, ui::window_number);
 
+    void map_invalidate_map_selection_tiles();
```

Comment:
These functions shouldn't be in input.h. I would suggest putting them in `map/tilemgr.h`.

Reply:
Sure, will do it.